src/applications/settings/user-settings/UserSettings.svelte

```diff
@@ -162,7 +162,7 @@
 
   .remaining-vertical-space {
     margin-right: -0.0625rem;
-    border-right: 0.0625rem solid var(--t5e-tab-strip-border-color);
+    border-right: 0.0625rem solid var(--t5e-component-tab-border);
```

Comment:
If I see changes to non-overhaul content after this point, I'll copy/paste the below phrase. I wasn't sure whether you meant to apply to these area, but just in case it was on accident, I'll tag them with a comment.

> Non-overhaul content here